### ISSUE TITLE
[Usability] Add tooltips to buttons

### ISSUE THREAD

**Issue by probonopd** [flaviotordini/minitube#230]
Please add tooltips to buttons.

For example, it is unclear what this button does as this is not a commonly used player control:

![image](https://user-images.githubusercontent.com/2480569/204124176-bd3e116d-8f76-4601-83e8-dcf1a13972ca.png)

**Comment by chrisgoldberg1** [comment]
This button has a tooltip (english as well as in german).

**Comment by probonopd** [comment]
I don't see one, but if it is there, then this can be closed. Maybe it's my theme or something like that why I don't see the tooltips.

**Comment by makkes** [comment]
I don't see it either (Arch Linux, minitube 3.9.3). Is there any way to debug this further to find out why it's not showing?

**Comment by chrisgoldberg1** [comment]
![2023-08-28_08-31](https://github.com/flaviotordini/minitube/assets/143316295/4e66c780-e4fe-44e7-a4c2-3283adfdd18e)

We speak about this, right? Or am I mistaken?
Using ArchLinux as well - works on original minitube and on my modified version (called: minitube-systray) till hopefully my pull requests are accepted.

**Comment by makkes** [comment]
> ![2023-08-28_08-31](https://user-images.githubusercontent.com/143316295/263612614-4e66c780-e4fe-44e7-a4c2-3283adfdd18e.png)
> 
> We speak about this, right?

Yes, exactly. Hovering over any of the buttons in that toolbar doesn't show tooltips for me. I'm using i3 but that shouldn't really affect the tooltips, should it?

**Comment by makkes** [comment]
As another data point, other applications linked with Qt 5 GUI such as keepassx do show tooltips.

**Comment by chrisgoldberg1** [comment]
No idea, but could be possible that the window manager interferes. I use xfce4 and it works flawless. But as you tell keepassx(c) tooltip works, I guess there is a wrong implementation. If I find time, I will take a look on it, but my engagement goes into another direction. I like to listen music while working and I missed a systray icon and the option to hide the application - so I implemented it.
